jsapp/js/account/accountFieldsEditor.component.tsx

```diff
@@ -150,7 +193,7 @@ export default function AccountFieldsEditor(props: AccountFieldsEditorProps) {
               name='country'
               type='outline'
               size='l'
-              isClearable
+              isClearable={!isFieldRequired('country')}
```

Comment:
This hides the 'closing X' when it's a required dropdown anyway — matches the designs better and looks cleaner